Modify systemd unit file reference to align with installation instruction (#6369)
matrix-org/synapse: contrib/systemd/README.md

# Setup Synapse with Systemd

> This is a setup for managing synapse with the user contributed systemd unit 

Reviewer: ```suggestion
This is a setup for managing synapse with a user contributed systemd unit 
```

file. It provides a `matrix-synapse` systemd unit file that should be tailored 

> to accomidate your installation in accordinance with the installation 

Reviewer: ```suggestion
to accommodate your installation in accordance with the installation 
```

> instructions provided in [installation instructions](https://github.com/matrix-org/synapse/blob/master/INSTALL.md).

Reviewer: You can actually just do this and Github/Gitlab will link it:

```suggestion
instructions provided in [installation instructions](../../INSTALL.md).
```

## Setup
1. Under the service section, ensure the `User` variable matches which user
you installed synapse under and wish to run it as. 

> 2. Under the service section, ensure the `WorkingDirectory` variablei matches

Reviewer: ```suggestion
2. Under the service section, ensure the `WorkingDirectory` variable matches
```

where you have installed synapse.
3. Under the service section, ensure the `ExecStart` variable matches the
appropriate locations of your installation.

> 4. Copy the `mastodon-synapse.service` to `/etc/systemd/system/`

Reviewer: ```suggestion
4. Copy the `matrix-synapse.service` to `/etc/systemd/system/`
```

:yum:

> 5. Start the service: `sudo systemctl start matrix-synapse`

Reviewer: ```suggestion
5. Start Synapse: `sudo systemctl start matrix-synapse`
```

> 6. Verify the service: `sudo systemctl status matrix-synapse`

Reviewer: ```suggestion
6. Verify Synapse is running: `sudo systemctl status matrix-synapse`
```

> 7. *optional* Enable the service at boot: `sudo systemctl enable matrix-synapse`

Reviewer: ```suggestion
7. *optional* Enable Synapse to start at system boot: `sudo systemctl enable matrix-synapse`
```
